sox.css

```
@@ -461,6 +461,7 @@
     float: right;
     position: sticky;
     top: 0;
+    z-index: 50;
```

Please could you add a comment here to say why we're setting the z-index? 

Something like this would do:
```suggestion
    /* z-index for compatibility with https://github.com/SmartManoj/SmartUserScripts/blob/master/SO_Lines.user.js */
    z-index: 50;
```

---

@SmartManoj you've marked this as resolved but I don't think you pushed the changes!

---

Now Its done